### ISSUE TITLE
Misaligned urlbar on boot

### ISSUE THREAD

**Issue by Piponja** [lingais/Zen-Browser-Super-Sleek-UI-Theme#3]
It seems that the urlbar is misaligned when opening the browser, although sometimes it does not happen but more often than not this is what happens:

![Captura de pantalla 2024-08-29 105015](https://github.com/user-attachments/assets/3ca41bfd-5d54-41dd-8f05-dbbc4008cb05)

A quick fix for this is to enter the customize toolbar mode, hover over the different density sizes (no need to actually switch between them since there is an animation that shows how each one looks like), and then exit that mode. That will align the urlbar temporarily until the browser is reopened:

![image](https://github.com/user-attachments/assets/346f6a25-63d2-447e-9ca0-2b6f75352bfe)

Zen 1.0.0-a.32 running on Windows 11 with just this mod installed to avoid any compatibility issues that could've caused this.

**Comment by onthejobtraining** [comment]
Yeah, tested on latest Zen 1.0.0-a.35 and It looks rather gnarly with compact mode enabled: 
![image](https://github.com/user-attachments/assets/05b88be7-75b0-48a3-94ab-e6239ba551d4)

This is without Super Sleek UI Theme: 
![image](https://github.com/user-attachments/assets/19d8f858-a10e-4fc8-aef7-b904c3e9afcf)

**Comment by paulwyszynski** [comment]
Noticed the same. Any fix planned for this?
